ui/src/app/views/workspace-list/component.ts

```diff
@@ -101,4 +111,15 @@ export class WorkspaceListComponent implements OnInit, OnDestroy {
   addWorkspace(): void {
     this.router.navigate(['workspace/build'], {relativeTo : this.route});
   }
+
+  get twoFactorBannerEnabled() {
+    if (this.firstSignIn == null) {
+      return false;
+    }
+    // If it is three days or more after they first sign in, deactivate the banner
```

Comment:
Have you confirmed this time limit with @jmandel ? I was just pulling this number from memory and I cannot personally vouch for its correctness.